config/samples/druid_v1alpha1_etcd.yaml

```diff
@@ -29,7 +29,7 @@ spec:
         role: test
     etcd:
         metrics: basic
-        image: eu.gcr.io/gardener-project/gardener/etcd:v3.4.13
+        image: eu.gcr.io/gardener-project/gardener/etcd:v3.4.13-bootstrap
```

Comment:
Shouldn't the [`etcdbrctl` image version](https://github.com/gardener/etcd-druid/blob/ba48ea629e59585d1b71060c9fd91c8778a60ad6/config/samples/druid_v1alpha1_etcd.yaml#L40) be changed too?

Reply:
Yes, will make this change.